distributed/worker.py

```diff
@@ -974,6 +974,7 @@ async def get_metrics(self) -> dict:
                 "memory": spilled_memory,
                 "disk": spilled_disk,
             },
+            comm_reserved_bytes=self.state.comm_nbytes,
```

Comment:
Worth noting that on line 967 above,

```
in_flight=self.state.in_flight_tasks_count,
```
This is also strictly the number of *incoming* in-flight tasks.

Reply:
Outgoing traffic can be measured:

- `Worker.outgoing_current_count` for number of outgoing connections
- there is no measure for outgoing number of bytes. However it's straightforward to add it to `Worker.get_data`:
https://github.com/dask/distributed/blob/4cf9bafdd8ce6524c9e9667f544880c8e561c645/distributed/worker.py#L1688
(add to a total counter after the above line and remove the same amount in a `finally` clause)

Reply:
To clarify: it's OK to leave metering of outgoing traffic to a different PR.
However, within the scope of this PR, make it clear through naming that the "in flight tasks" and nbytes are specifically about incoming traffic.

Reply:
Added `transfer_outgoing_bytes`

Reply:
Regarding `in_flight`: Can we safely rename this?